kotlin-ktor/build.gradle.kts

```
@@ -0,0 +1,48 @@
+import com.expediagroup.graphql.plugin.gradle.config.GraphQLSerializer
+
+description = "Example usage of Gradle plugin to generate GraphQL Kotlin Client"
+
+val expediaGroupVersion = "7.0.1"
```

Rename to match the actual library name GraphQL Kotlin, something like:
```suggestion
val graphQLKotlinVersion = "7.0.1"
```

---

✅